data/language/en-US.txt

```diff
@@ -2250,7 +2249,7 @@ STR_5730    :Nearest neighbor
 STR_5818    :{SMALLFONT}{BLACK}[Requires hardware display]{NEWLINE}Use nearest neighbor scaling{NEWLINE}when window scaling factor set{NEWLINE}to integer values (1, 2, 3, etc)
 STR_5821    :{SMALLFONT}{BLACK}Changes the color of the construction marker when building rides,{NEWLINE}paths, shops, scenery, etc.
 STR_5827    :{SMALLFONT}{BLACK}Sets the color scheme used for the GUI
-STR_5907    :{SMALLFONT}{BLACK}When enabled, zooming in will center around the cursor, as opposed to the screen center.
+STR_5907    :{SMALLFONT}{BLACK}When enabled, zooming in will center around the cursor, as opposed to the screen center. 
```

Comment:
Please revert the two changes to file: the trailing whitespace on this line and the removed newline on line 2238.